pkg/operator/workloadcontroller/workload_controller.go

```diff
@@ -85,6 +94,17 @@ func NewWorkloadController(
 		queue: workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "OpenShiftAPIServerOperator"),
 	}
 
+	// This function supports determining the number of master nodes
+	// to allow setting the deployment replica count to match.
+	c.countNodes = func(nodeSelector map[string]string) (*int32, error) {
```

Comment:
Too big for an anonymous function. Make this an actual function on the `OpenshfitAPIServerOperator`. Then assign this function to `countNodes` here.

Reply:
Done